libs/features/mas/web-components/src/variants/catalog.js

```diff
@@ -0,0 +1,97 @@
+import { VariantLayout } from './variant-layout.js';
+import { html, css } from 'lit';
+import { isMobileOrTablet } from '../utils.js';
+import { EVENT_MERCH_CARD_ACTION_MENU_TOGGLE } from '../constants.js';
+import { CSS } from './catalog.css.js';
+
+export class Catalog extends VariantLayout {
+  constructor(card) {
+    super(card);
```

Comment:
do we need this constructor?

Reply:
yes we do! the constructor adds the style if not there

Reply:
but when you don't declare, it should call the one from VariantLayout no?

Reply:
i wanted to shout NO, but i tried and yes it works :) sorry to be such a java guy :)